src/components/overview/ImageGallery/ImageGallery.jsx

```diff
@@ -2,12 +2,106 @@ import React, { useState, useEffect } from 'react';
 import DefaultView from './DefaultView.jsx';
 
 const ImageGallery = ({state, updateState}) => {
+  const [ currentStylePhotos, updateStylePhotos ] = useState([]);
+  const [ clickedImage, updateClickedImage ] = useState(0);
+
+  //update main image//
+  const updateMainImage = (event) => {
+    event.preventDefault();
+    let clickedImg = event.target.src;
```

Comment:
I'm pretty sure most of these could be `const`.

Reply:
you're right! It is just our of habit. I definitely need to use const as much as possible.
Thanks for letting me know!